code/modules/library/lib_machines.dm

```diff
@@ -520,6 +521,7 @@ datum/borrowbook // Datum used to keep track of who has borrowed what when and f
  */
 /obj/machinery/bookbinder
 	name = "Book Binder"
+	desc = "Ties up a sheaf or paper into a nice book format."

```

Comment:
"Sheaf or paper"? I'm not sure Sheaf is the right word here?

Reply:
Meant to be `sheaf of`